src/js/script.js

```diff
@@ -0,0 +1,63 @@
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, ''];
+let indexOfVoid = 15;
+let winCombination1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+let winCombination2 = [1, 5, 9, 13, 2, 6, 10, 14, 3, 7, 11, 15, 4, 8, 12];
+
+function shuffle() {
+    let mySet = new Set();
+
+    function randomInteger(min, max) {
```

Comment:
No need for function declaration here
You could define it outside. Or use variable (let/var/const)
var randomInteger = function(min, max)...

You have (max + 1 - min) inside but below you pass '1' as 'min'
(1 - 1) always gives 0 :)